.github/infra/cloudbuild-configure-triggers.yaml

```diff
@@ -17,12 +17,17 @@
 
 steps:
 - name: 'google/cloud-sdk:slim'
-  id: Configure build triggers deployment
+  id: Configure build triggers
   entrypoint: /bin/bash
   args:
   - -c
   - |
     set -o errexit
     set -o xtrace
-    gcloud beta deployment-manager deployments create cloud-build-triggers --config .github/infra/build-triggers.yaml || \
-      gcloud beta deployment-manager deployments update cloud-build-triggers --config .github/infra/build-triggers.yaml
+    for f in .github/infra/triggers/*.yaml
+    do
+      trigger_filename=$${f##*/}
+      trigger_name=$${trigger_filename%.*} 
+      gcloud beta builds triggers delete --quiet $${trigger_name} || true
+      gcloud beta builds triggers import --source $$f
```

Comment:
What project do the gcloud commands apply to? / How is that set?

Reply:
gcloud auth and project config is automatically handled by Cloud Build. When the project is not explicitly specified, Cloud Build will default to the same project that the build itself is running in.